pyproject.toml

```diff
@@ -93,6 +93,7 @@ structlog = { version = "*", optional = true }
 typing-extensions = "*"
 uvicorn = {extras = ["standard"], version = "^0.21.1", optional = true}
 importlib-resources = { version="^5.12.0", python = "<3.9" }
+prometheus-client = "^0.16.0"
```

Comment:
This should be an optional dependency.  I think it's probably best to create a `prometheus` group or something similar to install the additional dependency.

Reply:
> This should be an optional dependency. I think it's probably best to create a `prometheus` group or something similar to install the additional dependency.

Hi @cofin sorry did not realized that, Is it ok to add it in opentelemetry group in tools.poetry.extras or it is best to create a new one.

Reply:
I would create a new one.  This will make it a totally optional component that can be installed without OTEL.